src/core/layout/gel-defaults.js

```diff
@@ -190,7 +190,11 @@ export const config = screen => {
             id: "play",
             channel: buttonsChannel(screen),
             action: ({ screen }) => {
-                getScreenBelow(screen).scene.resume();
+                const screenBelow = getScreenBelow(screen);
+                screenBelow.scene.resume();
+                if (screenBelow._layout.buttons.achievementsSmall) {
+                    screenBelow._layout.buttons.achievementsSmall.setIndicator();
+                }
```

Comment:
We can optional chain now btw. Since it's in the latest Safari.

So this could just be `screenBelow._layout.buttons.achievementsSmall?.setIndicator()`